### PR TITLE
Fix sensors and TPM

### PR DESCRIPTION
**Pull request by SergiiDmytruk**
This intends to fix https://github.com/Dasharo/dasharo-issues/issues/415 and https://github.com/Dasharo/dasharo-issues/issues/416.

### CODE REVIEW

**Review by krystian-hebel — APPROVED** [review]

**Comment by tlaurion** [comment]
Fan speed is fixed! But not TPM issue.

File: payloads/external/skiboot/Kconfig [hunk 1]
```diff
@@ -15,7 +15,7 @@ config SKIBOOT_GIT_REPO
 config SKIBOOT_REVISION
 	string "Revision of skiboot payload"
 	default "d93ddbd39b4eeac0bc11dacbdadea76df2996c13" if BOARD_EMULATION_QEMU_POWER9
-	default "fa060c2c98b6eefde06a8bd78d1d8096c2bede37" if BOARD_RAPTOR_CS_TALOS_2
+	default "726443e2fce0c687074d01a68175942e7ae61ff2" if BOARD_RAPTOR_CS_TALOS_2
```
Comment (krystian-hebel) [inline payloads/external/skiboot/Kconfig:18]: See https://github.com/Dasharo/skiboot/pull/4#discussion_r1218181716

**Review by krystian-hebel — COMMENTED** [review]
Other than this one issue LGTM

File: payloads/external/skiboot/Kconfig (hunk 1 above)
Comment (SergiiDmytruk) [reply payloads/external/skiboot/Kconfig]: Changed commit hash to pick up only TPM driver fix.

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**